app/javascript/styles/call_list.scss

```diff
@@ -18,3 +18,25 @@
 .bootstrap_toggle_spacing .checkbox {
   margin-top: 0px;
 }
+
+// For autosortable columns, make the cursor a pointer
```

Comment:
Calls seemed to be half call-list/dashboard stuff and half call modal stuff. Have moved the call list stuff into here to reflect that.